test/AutoRest.TestServer.Tests/url/Generated/Operations/PathsOperations.cs

```diff
@@ -45,7 +45,7 @@ public static async ValueTask<Response> GetBooleanFalseAsync(ClientDiagnostics c
                 request.Method = RequestMethod.Get;
                 request.Uri.Reset(new Uri($"{host}"));
                 request.Uri.AppendPath("/paths/bool/false/", false);
-                request.Uri.AppendPath("false", false);
+                request.Uri.AppendPath("false", true);
```

Comment:
Eventually we would be doing escaping for constants as part of generation not yet tho.